From 148d89f0b3a652acc54b1a21807ffd6d0e81e279 Mon Sep 17 00:00:00 2001
From: Keith Packard <keithp@keithp.com>
Date: Sun, 15 Dec 2013 08:27:09 -0800
Subject: [PATCH] Replace 'pointer' with explicit 'void *'

To get rid of the 'pointer' typedef, stop using it locally. That way,
when _XTYPEDEF_POINTER is defined before Xdefs.h is included, it won't
get defined and Xdefs.h will still compile.

Signed-off-by: Keith Packard <keithp@keithp.com>
Reviewed-by: Alan Coopersmith <alan.coopersmith@oracle.com>
---
 Xdefs.h | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/Xdefs.h b/Xdefs.h
index 46ffdad..e25a208 100644
--- a/Xdefs.h
+++ b/Xdefs.h
@@ -101,8 +101,8 @@ typedef FSID AccContext;
 typedef struct timeval **OSTimePtr;
 
 
-typedef void (* BlockHandlerProcPtr)(pointer /* blockData */,
+typedef void (* BlockHandlerProcPtr)(void * /* blockData */,
 				     OSTimePtr /* pTimeout */,
-				     pointer /* pReadmask */);
+				     void * /* pReadmask */);
 
 #endif
-- 
1.8.4.2

